egs/aishell/s10b/local/convert_text_to_labels.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
+
+# Copyright 2020 Mobvoi AI Lab, Beijing, China (author: Fangjun Kuang)
+# Apache 2.0
+
+import argparse
+import os
+
+import kaldi
+
+
+def get_args():
+    parser = argparse.ArgumentParser(description='convert text to labels')
+
+    parser.add_argument('--lexicon-filename', dest='lexicon_filename', type=str)
+    parser.add_argument('--tokens-filename', dest='tokens_filename', type=str)
+    parser.add_argument('--dir', help='input/output dir', type=str)
+
+    args = parser.parse_args()
+
+    assert os.path.isfile(args.lexicon_filename)
+    assert os.path.isfile(args.tokens_filename)
+    assert os.path.isfile(os.path.join(args.dir, 'text'))
```

Comment:
There needs to be much more clarity on what this program is supposed to do, either as an extended
command or in the usage message.  It needs to be clear what the file formats are, for instance.
An example might be the easiest way.

Reply:
I will write example usage in the documentation to illustrate the purpose of the program.

Reply:
Fixed.